vsphere/vsphere.go

```diff
@@ -796,6 +811,8 @@ func recommendDatastore(
 		PodSelectionSpec: podSelectionSpec,
 		ConfigSpec:       spec,
 		ResourcePool:     &resourcePoolRef,
+		// lock the recommended resources for us for the next 90 seconds
+		ResourceLeaseDurationSec: 90,
```

Comment:
it seems like this var is not used anywhere

Reply:
What happens if there is a datastore cluster with a single datastore? Will the other nodes have to wait for 90s until they get a recommendation for that datastore? 
Also why 90?